#!/usr/bin/env python
PACKAGE = "morse_hanp_bridge"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("restrain_to_robot_FOV", bool_t, 0, "Restrain the detected humans to the robot field of view", False)
gen.add("robot_FOV_angle", double_t, 0, "The angle of the robot circular sector field of view (radians)", 1.57, 0, 6.2832)
gen.add("robot_FOV_range", double_t, 0, "The maximum distance seen by the circular sector field of view of the robot", 20, 0, 1000)

exit(gen.generate(PACKAGE, "morse_hanp_bridge", "morse_hanp_bridge"))